src/node_contextify.cc

```diff
@@ -634,7 +641,7 @@ class ContextifyScript : public BaseObject {
     Local<Value> arrow;
     if (!(maybe_value.ToLocal(&arrow) &&
           arrow->IsString() &&
-          stack->IsString())) {
+          !stack.IsEmpty() && stack->IsString())) {
```

Comment:
I'd maybe break this out into separate if statements for clarity (double negation is hard to parse for humans) but I'll leave that up to you.